lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -817,6 +818,23 @@ get_axis_communications_supplemental_authenticity(void *handle,
 
   return status;
 }
+/**
+ * Retrieves the certificate chain from the vendor handle. */
+const char *
+get_certificate_chain(signed_video_t *sv)
```

Comment:
The only member in `sv` that is used is `vendor_handle`. Then I think the input should be the handle. See for example `sv_vendor_axis_communications_teardown()`